framework/generated/khronos_generators/khronos_base_generator.py

```diff
@@ -340,10 +360,24 @@ def __init__(
 
 
         # Lower case prefix and structure type prefix for every supported Khronos API
-        self.valid_khronos_supported_api_prefixes = [
-            { 'vk', 'VK_STRUCTURE_TYPE_' },
-            { 'xr', 'XR_TYPE_' },
-        ]
+        # TODO: Eventually, we should move this info into a data file that we read (JSON?)
+        self.valid_khronos_supported_api_prefixes = []
+        self.valid_khronos_supported_api_prefixes.append(
+            ApiPrefix(
+                api_name='Vulkan',
+                command_prefix='vk',
+                struct_prefix='Vk',
+                struct_type_prefix='VK_STRUCTURE_TYPE_'
+            )
+        )
+        self.valid_khronos_supported_api_prefixes.append(
+            ApiPrefix(
+                api_name='OpenXR',
+                command_prefix='xr',
+                struct_prefix='Xr',
+                struct_type_prefix='XR_TYPE_'
+            )
+        )
```

Comment:
If we ever add other graphics bindings to XR we'll need to something for them here or?

Reply:
We can tweak this with whatever we need in the future.  And move it to a file we load it from.